gcsa/event.py

```diff
@@ -122,6 +122,8 @@ def assure_list(obj):
             self.end = start + timedelta(hours=1)
         elif isinstance(start, date):
             self.end = start + timedelta(days=1)
+        else:
+            self.end = end
```

Comment:
I'd move it to the first `if` statement:

```python
if end or start is None:
   self.end = end
elif ...
```

To DRY it and be explicit about the "both-None" case.